empress/xscape/costscape.py

```diff
@@ -31,6 +27,6 @@ def solve(newick_data, transferMin, transferMax, dupMin, dupMax, optional):
     print("Elapsed time %.2f seconds" % elapsedTime)
     plotcosts.plotcosts(CVlist, transferMin, transferMax, dupMin, dupMax, \
                         optional.outfile, \
-                        optional.log, display)
+                        optional.log, False)
```

Comment:
Let's go into `plotcosts.plotcosts` and remove the display argument from the code.